osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/resources/osgp-adapter-protocol-dlms.properties

```diff
@@ -1,3 +1,14 @@
+soapclient.use.client.auth=true
+#soapclient.default-uri=http://localhost:8080/osgp-secret-management/ws/SecretManagement
+soapclient.default-uri=https://localhost:443/osgp-secret-management/ws/SecretManagement
```

Comment:
Is this SOAP client sending messages to an HTTPD instance? Do we need some configuration for this?